src/server/verification/faceRecognition/faceRecognitionHelper.js

```diff
@@ -78,7 +78,8 @@ const Helper = {
   async isLivenessPassed(zoomData: FormData) {
     try {
       let res = await ZoomClient.liveness(zoomData)
-      log.debug('liveness result:', { res })
+      let results = res.data
+      results && log.debug('liveness result:', { results })
```

Comment:
results && is not required

Reply:
removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed